web/src/compose_closed_ui.js

```diff
@@ -144,19 +141,13 @@ export function update_buttons_for_private() {
 
 export function update_buttons_for_stream_views() {
     const text_stream = $t({defaultMessage: "Start new conversation"});
-    $("#new_conversation_button").attr(
-        "data-tooltip-template-id",
-        "new_topic_message_button_tooltip_template",
-    );
+    $("#new_conversation_button").attr("data-conversation-type", "stream");
     update_buttons(text_stream);
 }
 
 export function update_buttons_for_non_stream_views() {
     const text_stream = $t({defaultMessage: "Start new conversation"});
-    $("#new_conversation_button").attr(
-        "data-tooltip-template-id",
-        "new_stream_message_button_tooltip_template",
-    );
+    $("#new_conversation_button").attr("data-conversation-type", "non-stream");
```

Comment:
Hmm, `non-stream` is a bit misleading, since that sounds like it could be direct messages. Maybe `non-specific` would be a value that appropriately captures that we don't know what type of message to guess?

Reply:
Would this renaming also include the container function "update_buttons_for_**non_stream**_views()" as well?

Reply:
It made sense to rename the function name as well, since we don't really use "non_stream" outside of the closed composed UI related codes.

Reply:
Addressed by renaming "non-stream" to "non-specific" and "update_buttons_for_non_stream_views" to "update_buttons_for_non_specific_views".